apps/blog/src/posts/macos-pwa-install/macos-pwa-install.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: Web Apps on macOS Sonoma got a proper install experience
+excerpt: This post guides us how we detected Web Apps availability on macOS platform.
+description: How we enabled Sonoma's Web Apps support for PWA Install component.
+date: 2024-02-01
+updatedDate: 2024-02-01
+trending: true
+featured: true
+image: posts/macos-pwa-install/pic-1-wide.jpg
+isPost: true
+backUrl: '/'
+author:
+  name: Gleb Khmyznikov
+  twitter: khmyznikov
+  title: Software Engineer
+  image: /author_images/gleb_image.jpg
+  tagline: Eastern Europe based Software Engineer who loves PC hardware, gaming handhelds, classic cars and web technologies.
+tags:
+  - post
+  - macOS
+  - WebApps
+---
+
+<figure>
+  <video preload="none" controls poster="/posts/macos-pwa-install/pic-1-wide.jpg">
+    <source src="/posts/macos-pwa-install/video.webm" type="video/webm">
+  </video>
+  <figcaption>Installing PWA on macOS Safari</figcaption>
+</figure>
+
+### Backstory
+
+Traditionally for Apple, the company has been slow to adopt new web technologies. However, with the release of macOS Sonoma, Apple has made a big step forward in supporting Web Apps in general and Progressive Web Applications specifically. This is great news for developers and users alike, as it means that PWAs will be more accessible on Mac than ever before.
+
+But, also traditionally for Apple, there is no built-in browsers experiences for prompting PWA installation on Safari. This is where the [pwa-install](https://github.com/khmyznikov/pwa-install) component comes in. It's a simple, lightweight, framework agnostic web-component that provides a native-like installation experience for PWAs on iOS/iPadOS and now on macOS Sonoma. It's easy to use and works with any PWA or simple Web App, so you can get started right away.
+
+![PWA Install Instructions](/posts/macos-pwa-install/pic-3.jpg)
+
+Why easy installation matters? Progressive Web Applications offer a lot of flexibility and user engagement. One of the main selling points is their app-like presence on your device. However, not every browser has made it easy to install them. This update is set to make the installation of PWAs feel more native, especially for users on macOS Safari.
+
+### Implementation
+
+But how it was archived? Safari on macOS Sonoma supports the Web App Manifest, which is a JSON file that describes the Web App metadata. This includes the app's name, description, icons, and more. The pwa-install component uses this information to create a native-like installation experience for PWAs on iOS/iPadOS and macOS Sonoma.
+
+But here's a catch: Safari on macOS don't provide any API to detect Web Apps availability. So we had to use a workaround to detect Web Apps availability on macOS platform.
```

Comment:
```suggestion
But here's a catch: Safari on macOS doesn't provide any API to detect Web Apps availability. So we had to use a workaround to detect Web Apps availability on macOS platform.
```

Reply:
I am confused here about what "availability" means. Is it availability for install?

Reply:
Yeah it's about installability